extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -112,7 +112,9 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				await this.createAndAddBookModel(bookPath, !!isNotebook);
```

Comment:
>this.createAndAddBookModel [](start = 10, length = 26)

createAndAddBookModel seems to handle the reveal part as well and looks like the lines below it are redundant.